ui/src/App.tsx

```diff
@@ -64,34 +64,50 @@ function App() {
           </Stack>
         </Toolbar>
       </AppBar>
-      <Container sx={{ py: 2, flex: 1 }}>
-        <Typography variant="h4">Hello</Typography>
-        <Typography variant="body1">Explanation stuff</Typography>
-        <Box
-          sx={theme => ({
-            display: 'grid',
-            gridTemplateColumns: '1fr 1fr 1fr',
-            gridAutoRows: '1fr',
-            width: '600px',
-            height: '600px',
-            border: '3px solid black',
-            borderRadius: theme.shape.borderRadius,
-            overflow: 'hidden',
-            '& > button': {
-              backgroundColor: 'grey',
-              borderRadius: "0px",
-              '&:not(:nth-child(3n))': {
-                borderRight: 'thin solid black',
-              },
-              '&:nth-child(-n+6)': {
-                borderBottom: 'thin solid black',
-              },
-            },
-          })}
-        >
-          {[...Array(9).keys()].map(entry => <GridButton key={entry}/>)}
-        </Box>
-      </Container>
+        <Container sx={{ py: 2, flex: 1}}>
+          <Typography variant="h4" sx = {{textAlign: 'center'}}>Hello</Typography>
+          <Typography variant="body1" sx = {{textAlign: 'center'}}>Explanation stuff</Typography>
+          <Stack alignItems='center'>
+            <Stack direction='row' alignItems={'center'}>
+              <Typography variant="body1" sx={{textAlign:'center', width:'200px'}}>A category</Typography>
+              <Typography variant="body1" sx={{textAlign:'center', width:'200px'}}>Some other category</Typography>
+              <Typography variant="body1" sx={{textAlign:'center', width:'200px'}}>Looooooooong looooooong multi-line category</Typography>
+            </Stack>
+            <Stack direction='row'>
+              <Stack width='200px' alignItems='center'>
+                <Typography height='200px' variant="body1" sx={{textAlign:'center', alignItems:'center', display:'flex'}}>Author</Typography>
+                <Typography height='200px' variant="body1" sx={{textAlign:'center', alignItems:'center', display:'flex'}}>Some other author with a super-duper multi-line name</Typography>
+                <Typography height='200px' variant="body1" sx={{textAlign:'center', alignItems:'center', display:'flex'}}>One more author</Typography>
```

Comment:
```suggestion
                <Typography height='200px' variant="body1" sx={{textAlign:'center', alignItems:'center', display:'flex'}}>Author</Typography>
                <Typography height='200px' variant="body1"}>Some other author with a super-duper multi-line name</Typography>
                <Typography height='200px' variant="body1">One more author</Typography>
```

Reply:
What is the purpose of this suggestion? Won't this remove the centering of the second and third authors' names?